src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -79,7 +79,7 @@
     private final String paymentReference;
     private final DynamicList applicantSolicitor1PbaAccounts;
     private final ClaimType claimType;
-    private final SuperClaimType superClaimType;
+    private SuperClaimType superClaimType;
```

Comment:
I'd need confirmation for this, but superClaimType is what says a claim is specified or not. I don't think this is something that can change afterwards

Reply:
superClaimType was set specifically for UNSPEC in CaseData build() method and due to that was not able to create test cases for SPEC(please check DefendantResponseApplicantNotificationHandlerTest.java) because of that. So the only solution I could find was making superClaimType not final and setting it as UNSPEC atStateClaimDraft().